NativeScript/runtime/Helpers.mm

```diff
@@ -691,3 +691,13 @@
     Log(@"%s", stack.c_str());
     assert(false);
 }
+
+void tns::StopExecutionAndLogStackTraceIfInDebug(v8::Isolate* isolate) {
```

Comment:
rename to `StopExecutionAndLogStackTrace` or remove

Reply:
Done